tac/agents/v1/base/actions.py

```diff
@@ -122,9 +122,9 @@ def unregister_service(self) -> None:
         :return: None
         """
         if self.game_instance.goods_demanded_description is not None:
-            self.out_box.out_queue.put(OutContainer(message_id=1, service_description=self.game_instance.goods_demanded_description))
+            self.out_box.out_queue.put(OutContainer(message_id=1, service_description=self.game_instance.goods_demanded_description, is_unregister=True))
```

Comment:
@marcofavorito  we never unregistered :o

Reply:
Thanks for spotting!